models/reviews.model.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../db/connection')
+
+exports.fetchReviewByID = (id) => {
+    if(!id){
+        return Promise.reject({status: 400, msg: "Invalid Id"})
+    }
+    return db.query(`SELECT * FROM reviews WHERE review_id = ${id}`)
+    .then(result => result.rows)
+    .then( review => {
```

Comment:
There's no need for 2 `.then` blocks. You can just return `result.rows[0]` in the first `.then `block